pkg/tar/tar.go

```diff
@@ -154,6 +163,19 @@ func NewParanoid(fs fs.FileSystem) Tar {
 	}
 }
 
+// NewParanoidWithTimeout creates a new Tar with the provided timeout extracting files.
+// It has restrictions on what it can do while extracting files.
+func NewParanoidWithTimeout(fs fs.FileSystem, timeout time.Duration) Tar {
```

Comment:
Added new constructors in case we need to increase the timeouts on the OpenShift side of things